### PR TITLE
Minimum merge operation in array to make it palindrome

### PR DESCRIPTION
**Pull request by Aarti002**
issue #780 solved
Please @tarun26091999 review and merge my code!
Thank you!

### CODE REVIEW

File: Array/Python/merge_to_palindrome.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
+#Ques: Find minimum number of merge operations required to make array palindrome
+#Language: Python 3
+
+#Basically, we are comparing array elements from both ends if they are equal then decrementing and incrementing both indexs otherwise we will merge elements of array which is minimum
+#input:                         ----->(n) size of array
+#                                       ---->in next input line user have to provide space seperated integers
```
Comment (Rohan27s) [inline Array/Python/merge_to_palindrome.py:6]: Remove the unnecessary space

File: Array/Python/merge_to_palindrome.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+if __name__=="__main__":
+    n=int(input('Enter size of array: '))
+    arr=list(map(int,input().strip().split(' ')))[:n]
+
+    print('Minimum merge operations to make this array palindrome: ',end="")
+    print(merge_op(arr, n))
+
+#Time complexity:O(n)
+
+#e.g-1)
```
Comment (Rohan27s) [inline Array/Python/merge_to_palindrome.py:51]: Write Test Case 1 instead of eg 1

**Review by Rohan27s — CHANGES_REQUESTED** [review]

**Comment by Aarti002** [comment]
@Rohan27s i have made required changes please can you review again.

**Comment by Rohan27s** [comment]
Resolve the conflicts

**Comment by Aarti002** [comment]
@Rohan27s i resolved it.
Thank you!

**Pull request merged**